lib/questions.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview all the questions for the quiz
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+export const langQuestions = [{
+    type: "multiselect",
+    name: "languages",
+    message: "What do you want to lint?",
+    choices: [
+        { message: "JavaScript", name: "javascript" },
+        { message: "JSON", name: "json" },
+        { message: "JSON with comments", name: "jsonc" },
+        { message: "JSON5", name: "json5" },
+        { message: "Markdown", name: "md" }
+    ],
+    initial: 0
+}];
+
+export const jsQuestions = [
+    {
+        type: "select",
+        name: "purpose",
+        message: "How would you like to use ESLint?",
+        initial: 1,
+        choices: [
+            { message: "To check syntax only", name: "syntax" },
+            { message: "To check syntax and find problems", name: "problems" }
+        ]
+    },
```

Comment:
Perhaps this question should be asked regardless of the language choice, and if the answer is just "syntax" then the created config shouldn't include recommended configs from `@eslint/json` and `@eslint/markdown`?

Reply:
👍 makes sense to me. will update it later.

Reply:
https://github.com/eslint/create-config/pull/151/commits/db143816f7e9fa94195792f838d8e2794a5d3542